Cartfile

```diff
@@ -2,5 +2,5 @@ github "Alamofire/Alamofire" == 3.1.2
 github "DaveWoodCom/XCGLogger" == 3.1.1
 github "Hearst-DD/ObjectMapper" == 1.0.1
 github "rhodgkins/SwiftHTTPStatusCodes" == 2.0.1
-github "daltoniam/Starscream" == 1.0.2
-github "tristanhimmelman/AlamofireObjectMapper" == 2.1.0
\ No newline at end of file
+github "daltoniam/Starscream" == 1.1.1
```

Comment:
@vherrin, @germanattanasio, @jsstylos, @mbouldin:

Do we need legal approval to update the release version of our dependencies?

(This PR updates Starscream from 1.0.2 to 1.1.1 and AlamofireObjectMapper from 2.1.0 to 2.1.3 -- these are both very important updates that fix build problems in the master branch.)